source/turtles.rst

```diff
@@ -157,7 +165,7 @@ Code in files
 
 Most code is written and executed from a file. 
 
-Using SublimeText create a file named `my_turtle_file.py` with this code:: 
+Using SublimeText to create a file named `my_turtle_file.py` with this code:: 
```

Comment:
The description of the ed centre according to Emma and T are here: 
https://docs.google.com/document/d/1_DGHcrD0RQ9LoLiMnxnMN63W83Xvf4rejGI3SHiNDIQ/edit?ts=5772ac53

Under the heading "Location - education centre"